docs/source/markdown/podman-container-checkpoint.1.md

```diff
@@ -56,11 +50,33 @@ This option must be used in combination with the **--export, -e** option.
 When this option is specified, the content of volumes associated with
 the container will not be included into the checkpoint tar.gz file.
 
+#### **--keep**, **-k**
+
+Keep all temporary log and statistics files created by CRIU during checkpointing. These files
+are not deleted if checkpointing fails for further debugging. If checkpointing succeeds these
+files are theoretically not needed, but if these files are needed Podman can keep the files
+for further analysis.
+
+#### **--latest**, **-l**
+
+Instead of providing the container name or ID, checkpoint the last created container. (This option is not available with the remote Podman client)
+
 #### **--pre-checkpoint**, **-P**
 
 Dump the container's memory information only, leaving the container running. Later
 operations will supersede prior dumps. It only works on runc 1.0-rc3 or higher.
 
+#### **--leave-running**, **-R**
```

Comment:
Could you alphabetize the options.  --leave-running should come before --pre-checkpoint.

Reply:
I missed that one. Thanks. Fixed.